### PR TITLE
Docs: Add email subject and body to site-example

### PR DESCRIPTION
**Pull request by rubo77**

### CODE REVIEW

**Comment by jplitza** [comment]
I like the idea. However, I think explaining how mailto-Links work doesn't make sense. Either there's a handler configured and people are used to clicking on them and will notice the information is already prefilled, or not, in which case they are probably used to simply copying the address.

Also, for consistency, you used spaces around the pubkey variable but not around the others.

**Comment by rubo77** [comment]
You think, the small text is obsolete?

**Comment by jplitza** [comment]
Exactly.

**Comment by rubo77** [comment]
The small hint looks like this: 

auswahl_045

**Comment by NeoRaider** [comment]
I don't like the small text either.

Also, this is missing escaping (`escape()` in the text, `urlescape()` in the URL).

In addition, the other languages need to be updated, at least English.

**Comment by rubo77** [comment]
I also added `urlescape(sysconfig.primary_mac)` don't know if escaping is needed here?

**Comment by NeoRaider** [comment]
The primary MAC doesn't need to be escaped.

**Comment by rubo77** [comment]
rebased to master

File: docs/site-example/i18n/de.po [hunk 1]
```diff
@@ -23,7 +23,9 @@ msgstr ""
 "kann sich dein Knoten mit dem Mesh-VPN dort verbinden. Bitte "
 "schicke dazu diesen Schlüssel und den Namen deines Knotens "
 "(<em><%=escape(hostname)%></em>) an "
-"<a href=\"mailto:keys@alpha-centauri.freifunk.net\">keys@alpha-centauri.freifunk.net</a>."
+"<a href=\"mailto:keys@alpha-centauri.freifunk.net?subject=Anmeldung:%20<%= urlescape(hostname) %>&amp;"
+"body=%23%20<%= urlescape(hostname) %>%0A%23%20<%= usysconfig.primary_mac %>%0Akey%20%22"
```
Comment (NeoRaider) [inline docs/site-example/i18n/de.po]: There's a typo here (usysconfig instead of sysconfig), in all three versions.

Comment (rubo77) [inline docs/site-example/i18n/de.po]: corrected.

**Comment by NeoRaider** [comment]
Another idea: how about enclosing the whole body of the mail in a single *urlescape* call? That would avoid all that unreadable percent-encoded text. The same could be done with the subject, but it's not as important there.

**Comment by rubo77** [comment]
that would look much cleaner, you can add that while merging? I have no way to test this before friday

**Comment by NeoRaider** [comment]
Merged. I changed it to use a single *urlencode*, and made the body match the other part of the example.

**Pull request closed without merging**